test/dfusionMultiSafes.js

```diff
@@ -29,22 +29,30 @@ const { checkCorrectnessOfDeposits } = require("../scripts/utils/price-utils")(w
 
 const { toErc20Units } = require("../scripts/utils/printing_tools")
 
+const TEN = new BN(10)
+
 const checkPricesOfBracketStrategy = async function(lowestLimit, highestLimit, bracketSafes, exchange) {
   const stepSizeAsMultiplier = Math.pow(highestLimit / lowestLimit, 1 / bracketSafes.length)
-  let multiplicator = new BN("10")
-  if ((lowestLimit + highestLimit) / 2 < 10) {
-    multiplicator = new BN("10000000")
-  }
+
   // Correctness assertions
   for (const [index, bracketAddress] of bracketSafes.entries()) {
+    let multiplicator = new BN("100")
+    if (lowestLimit * Math.pow(stepSizeAsMultiplier, index) < 10) {
+      multiplicator = new BN("1000000")
+    }
```

Comment:
I find these steps difficult to understand. I think this is for implementing floats with a fixed amount of decimals, and adapting the number of decimals depending on some parameter. Does it make sense to implement these floats for clarity?